src/Cms/PluginAssets.php

```diff
@@ -66,15 +66,14 @@ public static function resolve(string $pluginName, string $filename)
                 static::clean($pluginName);
 
                 $target = $plugin->mediaRoot() . '/' . $filename;
-                $url    = $plugin->mediaUrl() . '/' . $filename;
 
                 // create the plugin directory first
                 Dir::make($plugin->mediaRoot(), true);
```

Comment:
`F::link()` already create dir for target in above symlink line. I think we no longer need this line.

Reply:
Done https://github.com/getkirby/kirby/pull/3672/commits/c268f54ed39f3bb593c87c0d4a2271afa5e9d597